src/platform-implementation-js/dom-driver/gmail/widgets/gmail-mole-view-driver.ts

```diff
@@ -97,14 +97,32 @@ class GmailMoleViewDriver {
       }
     }
 
-    querySelector(this._element, '.inboxsdk__mole_view_content').appendChild(
+    querySelector(this.#element, '.inboxsdk__mole_view_content').appendChild(
       options.el,
     );
   }
 
   show() {
     const doShow = (moleParent: HTMLElement) => {
-      moleParent.insertBefore(this._element, last(moleParent.children)!);
+      const lastChild = last(moleParent.children)!;
+
+      if (lastChild instanceof HTMLElement) {
+        /**
+         * 2023-10-02 When Google Chat is enabled, we need to keep track of the `order` style property for moles added.
+         * If we don't, we end up with moles added on top of the sidebar because of a chat element in the same parent element as moleParent.
+         *
+         * This fix ratches the order of the element padding in order to present moles before the sidebar. Whether or not this fix works with Gmail's CSS in a RTL language is unknown.
+         */
+        const oldOrder = lastChild.style.order;
+        const oldOrderNum = parseInt(oldOrder, 10);
+
+        if (oldOrder && !isNaN(oldOrderNum)) {
+          this.#element.style.order = oldOrder;
+          lastChild.style.order = `${oldOrderNum - 1}`;
```

Comment:
Q: Does this mean we'll end up with duplicate order number elements?

Reply:
A: no, because the last child (the spacer on the right of the composes):

- starts at 0
- Gmail seems to only use positive integers for compose and chat moles `order` properties

The last bullet point means that this fix causes any mole that's added to always be to the right of any subsequently added compose. That probably dooms this fix.